apps/storybook/src/LabeledDropdown.stories.tsx

```diff
@@ -8,9 +8,38 @@ export default {
 };
 
 export const Normal = () => (
+  <LabeledDropdown
+    title={text('Title', 'Airport')}
+    subtitle={text('Subtitle', '(Optional)')}
+    options={[
+      { value: 'LHR', label: 'Heathrow' },
+      { value: 'LGW', label: 'Gatwick' },
+      { value: 'STN', label: 'Stansted' },
+      { value: 'LTN', label: 'Luton' },
+      { value: 'LCY', label: 'City' },
+      { value: 'SEN', label: 'Southend' },
+    ]}
+    value={select(
+      'Value',
+      {
+        Heathrow: 'LHR',
+        Gatwick: 'LGW',
+        Stansted: 'STN',
+        Luton: 'LTN',
+        City: 'LCY',
+        Southend: 'SEN',
+      },
+      'LHR',
+    )}
+    enabled={boolean('Enabled', true)}
+  />
+);
+
+export const Required = () => (
   <LabeledDropdown
```

Comment:
This story would ideally start with no option selected to show the default empty state without error.